src/data/calculate.ts

```diff
@@ -69,6 +71,21 @@ interface CalculatorResult {
   upperBound: number
 }
 
+const MAX_DELAY_FACTOR = 2
+
+export const DAY_0 = new Date(2020, 1, 12)
+const MS_PER_DAY = 1000 * 60 * 60 * 24
+
+// From https://covid19-projections.com/estimating-true-infections-revisited/
+const prevalanceRatio = (positivityPercent: number | null, date: Date) => {
+  const day_i = (date.getTime() - DAY_0.getTime()) / MS_PER_DAY
+  if (positivityPercent === null || positivityPercent > 100) {
+    positivityPercent = 100
+  }
```

Comment:
This is probably just a case of me not understanding, but it seems odd to me to have a null `positivityPercent` result in 100. It seems that the formula just results in "2" in that case. 

Maybe it'd be useful to add a comment to explain the intended result. Like "If we have no positivity data, we want to act as though... XYZ"

Reply:
Thanks!

Reply:
This is the same behavior as before - if there's no positive case percentage data, we assume the worst.

100% positivity rate -> `2 + 1250 / (day_i +25)`. Today, `day_i` is 300-something that's a little under 6x (so still less than the MINIMUM underreportingFactor previously.

In late summer (day 125 or so?) it would have been about 10x, which is what the underreportingFactor was for null positivityPercent (or positivityPercent over 15)